addons/web_editor/controllers/main.py

```diff
@@ -26,6 +27,47 @@
 logger = logging.getLogger(__name__)
 DEFAULT_LIBRARY_ENDPOINT = 'https://media-api.odoo.com'
 
+def generate_channel_signatures(env, model_name, field_name, res_id):
+    # Verify access to the edition channel.
+    if not env.user.has_group('base.group_user'):
+        raise AccessDenied()
+
+    rec = env[model_name].browse(res_id)
+
+    rec.check_access_rights('read')
+    rec.check_field_access_rights('read', [field_name])
+    rec.check_access_rule('read')
+    rec.check_access_rights('write')
+    rec.check_field_access_rights('write', [field_name])
+    rec.check_access_rule('write')
```

Comment:
As noted by @mart-e, this can fail on special models where the user does not have full access, but may still be able to edit an Html field. One example is the `res.users` model which allows users to edit their own preferences, despite having no `write` access on `res.users`.

The current code has the same problem, and causes all nightly builds to be red on runbot.

For these cases it is fine to degrade and prevent collaborative edition, but it should not crash -> `try/except` or use the "no-raise" variants of ACL checks.

Reply:
Done. Now, it prevent collaborative edition with the suggested `try/except`.